.ruff.toml

```diff
@@ -178,9 +178,6 @@ lint.ignore = [
     "TRY201",  # verbose-raise
     "TRY301",  # raise-within-try
 
-    # pyupgrade (UP)
-    "UP038",  # isinstance using union separators. The code is slower as of Python 3.11-3.12
```

Comment:
We don't need to ignore this explicitly anymore because it was deprecated in [Ruff 0.10.0](https://github.com/astral-sh/ruff/releases/tag/0.10.0).